src/EntryList.js

```diff
@@ -0,0 +1,12 @@
+import App from "./App";
```

Comment:
Do you need App here? You have imported App?

Reply:
I do not know 
Just I thought maybe I need it 
Because there are some codes in it belongs to some codes in App.js

Reply:
No, you don't need it, 
if there's a code in App.js and you need it here, then you should import that code.
You've imported App component which you don't need